jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -120,6 +124,12 @@ public Response registerInstallation(
         // otherwise we register a new installation:
         logger.finest("Mobile Application on device was launched");
 
+        //let's do update the analytics
+
+        String pushIdentifier = extractPushIdentifier(request);
```

Comment:
@sebastienblanc please move the `extractPushIdentifier()` to a util and let's name the header different (e.g. aerogear-notification-id) or something along this